frame/assets/src/functions.rs

```diff
@@ -103,16 +103,24 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 		Remove
 	}
 
+	/// Returns `true` when the balance of `account` can be increased by `amount`.
+	///
+	/// - `id`: The id of the asset that should be increased.
+	/// - `who`: The account of which the balance should be increased.
+	/// - `amount`: The amount by which the balance should be increased.
+	/// - `will_increase_supply`: Will the supply of the asset be increased by `amount` at the same
```

Comment:
```suggestion
	/// - `increase_supply`: Will the supply of the asset be increased by `amount` at the same
```

or update the variable below